src/github.com/mozilla-services/pushgo/simplepush/structs.go

```diff
@@ -42,8 +40,8 @@ type PushCommand struct {
 
 type PushWS struct {
 	uaidLock sync.RWMutex
```

Comment:
Noting for the future: now that we're using a deadline instead of `time.AfterFunc` for the handshake timeout, I think we can remove the mutex around `uaid`.